includes/template-library/sources/local.php

```diff
@@ -1173,7 +1173,7 @@ public function admin_export_multiple_templates( $redirect_to, $action, $post_id
 	 */
 	public function admin_print_tabs( $views ) {
 		//phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce is not required to retrieve the value.
-		$current_type = Utils::get_super_global_value( $_REQUEST, self::TAXONOMY_TYPE_SLUG ) ?? '';
+		$current_type = Utils::get_super_global_value( $_REQUEST, self::TAXONOMY_TYPE_SLUG, '' );
```

Comment:
We want to deprecate the default value parameter
```suggestion
		$current_type = Utils::get_super_global_value( $_REQUEST, self::TAXONOMY_TYPE_SLUG ) ?? '';
```

Reply:
Added back.